api/docs/docs.go

```diff
@@ -32,7 +32,380 @@ var doc = `{
+                            }
+                        },
+                        "headers": {
+                            "Token": {
+                                "type": "string",
+                                "description": "qwerty"
+                            }
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "#/definitions/api.HTTPError"
+                        }
+                    }
+                }
+            }
+        },
+        "/receipt/{tx}": {
+            "get": {
+                "description": "Get receipt by tx hash, used for cross-chain transfer",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "summary": "Get receipt by tx hash",
+                "operationId": "get-receipt-by-tx",
+                "parameters": [
+                    {
+                        "type": "string",
+                        "description": "tx hash",
+                        "name": "tx",
+                        "in": "path",
+                        "required": true
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/core.GetReceiptResp"
+                            }
+                        },
+                        "headers": {
+                            "Token": {
+                                "type": "string",
+                                "description": "qwerty"
+                            }
+                        }
+                    },
+                    "400": {
+                        "description": "Bad Request",
+                        "schema": {
+                            "$ref": "#/definitions/api.HTTPError"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "api.HTTPError": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "type": "integer",
+                    "example": 400
+                },
+                "message": {
+                    "type": "string",
+                    "example": "status bad request"
+                }
+            }
+        },
+        "core.GetEthHeaderWithProofJSONResp": {
+            "type": "object",
+            "properties": {
+                "eth_header": {
+                    "type": "object",
+                    "$ref": "#/definitions/eth.DarwiniaEthHeaderHexFormat"
+                },
+                "mmr_proof": {
```

Comment:
use mmr_root

Reply:
This is actually mmr_proof, `[]string`, the `mmr_root` is `root` in this pr, this field is just for test for now.

Reply:
I think this field is ok.